### PR TITLE
add ability to navigate through users preview in search using arrows

### LINKED ISSUE #367: Set up down & up arrow key bindings with user autocomplete (search on home page & search to add collaborator)
image

### PR DESCRIPTION
**Pull request by stushurik**
Closes datavis-tech/vizhub-issue-tracker/issues/367

### CODE REVIEW

File: packages/neoFrontend/src/styles.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ export const Clickable = styled.div`
   &:active {
     background-color: ${active};
   }
+
+  &.active {
```
Comment (curran) [inline packages/neoFrontend/src/styles.js:26]: Looks like duplicated CSS.

**Review by curran — APPROVED** [review]
Looks great!

File: packages/neoFrontend/src/styles.js (hunk 1 above)
Comment (curran) [reply packages/neoFrontend/src/styles.js:26]: Removed duplicated CSS in `master`.

Comment (stushurik) [reply packages/neoFrontend/src/styles.js:26]: @curran after your removed it, options are not highlighted when you navigate, we need bring back that

Comment (curran) [reply packages/neoFrontend/src/styles.js:26]: Why is it like that? Isn't it the exact same CSS twice?

Comment (curran) [reply packages/neoFrontend/src/styles.js:26]: Oh sorry - I see now that it's `:` vs `.`. Will put it back.

Comment (curran) [reply packages/neoFrontend/src/styles.js:26]: Added back https://github.com/curran/vizhub/commit/b142f097ce0340e61c86e30fe9333f5c6b5aec63

Comment (curran) [reply packages/neoFrontend/src/styles.js:26]: Deployed.

**Pull request merged**